controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/deployment/ApplicationStore.java

```diff
@@ -37,4 +37,10 @@
     /** Removes all tester packages for the given tester. */
     void removeAll(TesterId tester);
 
+    /** Stores the given application package as the development package for the given application. */
+    void putDev(ApplicationId application, byte[] applicationPackage);
```

Comment:
Will there be a separate method for `perf` or maybe rename this more generally and add extra arg?

Reply:
Both are development things, though ...

Reply:
I would like to keep it just one thing. Simpler, less stuff :)

Reply:
Sure, I'm just thinking about the implementation of this method, so we don't override the `dev` package when someone deploys to `perf`...